src/charm.py

```diff
@@ -265,42 +317,54 @@ def _get_gnb_name_tac_list(self) -> List[Tuple[str, int]]:
             gnb_name = gnb_identity_relation.data[gnb_identity_relation.app].get("gnb_name", "")
             gnb_tac = gnb_identity_relation.data[gnb_identity_relation.app].get("tac", "")
             if gnb_name and gnb_tac:
-                gnb_name_tac_list.append((gnb_name, int(gnb_tac)))
+                gnb_name_tac_list.append(GnodeB(name=gnb_name, tac=int(gnb_tac)))
         return gnb_name_tac_list
 
-    def _get_upf_config(self) -> str:
-        """Get the UPF configuration for the NMS in a json.
+    def _sync_gnbs(self, webui_gnbs: List[GnodeB], relation_gnbs: List[GnodeB]) -> None:
+        """Align the gNBs from the `fiveg_gnb_identity`relations with the ones in webui."""
+        relation_gnb_names = {gnb.name for gnb in relation_gnbs}
```

Comment:
Why the need for this dict? GnodeB's are dataclasses and should be comparable directly

Reply:
I was over complexifying the method thinking about the case in which a gnb modifies its tac. 
Now it's much simpler